Source: philantrop@exherbo.org
Upstream: No, they work on the upcoming 2.0 only.
Reason: Has more automagic deps than legitimate ones... 

diff -Naur ulogd-1.24.orig/configure.in ulogd-1.24/configure.in
--- ulogd-1.24.orig/configure.in	2005-11-25 20:58:27.000000000 +0100
+++ ulogd-1.24/configure.in	2008-09-26 23:31:52.000000000 +0200
@@ -9,7 +9,6 @@
 
 dnl Checks for libraries.
 AC_CHECK_LIB(dl, dlopen)
-AC_CHECK_HEADER(pcap.h,HAVE_PCAP_H=true)
 
 dnl Checks for header files.
 AC_HEADER_DIRENT
@@ -34,14 +33,14 @@
 dnl
 dnl test for MySQL
 dnl
-AC_ARG_WITH(mysql,
- --with-mysql=<directory>		mysql installed in <directory>,[
-if test $withval != yes
-then
+AC_ARG_WITH(mysql, AS_HELP_STRING([--with-mysql=<directory>],[mysql installed in <directory>]),[with_mysql=$withval
+],[with_mysql=no])
+if test $withval != yes -a $withval != no ; then
         dir=$withval
 else
         dir="/usr/local"
 fi
+if test $withval != no; then
 mysqldir=""
 AC_MSG_CHECKING(for MySQL files)
 for d in $dir/bin /usr/bin /usr/local/bin /usr/local/mysql/bin /opt/mysql/bin /opt/packages/mysql/bin
@@ -94,8 +93,7 @@
 	fi
 
 fi      
-])      
-
+fi
 
 dnl
 dnl Check whether the user wants log IP-addresses as strings rather
@@ -116,14 +114,16 @@
 dnl
 dnl test for PostgreSQL
 dnl
-AC_ARG_WITH(pgsql,
- --with-pgsql=<directory>		pgsql installed in <directory>,[
-if test $withval != yes
+
+AC_ARG_WITH(pgsql, AS_HELP_STRING([--with-pgsql=<directory>],[pgsql installed in <directory>]),[with_pgsql=$withval],[with_pgsql=no])
+if test $withval != yes -a $withval != no
 then
         dir=$withval
 else
         dir="/usr/local"
 fi
+if test $withval = yes
+then
 pgsqldir=""
 AC_MSG_CHECKING(for PGSQL files)
 for d in $dir/bin /usr/bin /usr/local/bin /usr/local/pgsql/bin /opt/pgsql/bin /opt/packages/pgsql/bin
@@ -154,7 +154,7 @@
 	AC_SUBST(PGSQL_LIB)
 
 fi      
-])      
+fi      
 
 dnl
 dnl Check whether the user wants to log IP-addresses as strings rather
@@ -171,15 +171,16 @@
 dnl
 dnl test for sqlite3
 dnl
-AC_ARG_WITH(sqlite3,
- --with-sqlite3=<directory>		sqlite3 installed in <directory>,[
-if test $withval != yes
+AC_ARG_WITH(sqlite3, AS_HELP_STRING([--with-sqlite3=<directory>],[sqlite3 installed in <directory>]),[with_sqlite3=$withval],[with_sqlite3=no])
+if test $withval != yes -a $withval != no
 then
         dir=$withval
 else
         dir="/usr/local"
 fi
-mysqldir=""
+if test $withval = yes
+then
+sqlite3dir=""
 AC_MSG_CHECKING(for sqlite3 files)
 for d in $dir /usr /usr/local /usr/local/sqlite3
 do
@@ -232,7 +233,7 @@
 	AC_SUBST(SQLITE3_LIB)
 
 fi      
-])      
+fi      
 
 
 dnl
@@ -251,6 +252,17 @@
 ])
 
 
+dnl
+dnl test for libpcap
+dnl
+HAVE_PCAP_H=""
+AC_ARG_WITH([pcap], AS_HELP_STRING([--without-pcap], [Build without pcap library (default: test)]))
+
+if test "x$with_pcap" != "xno"; then
+   AC_CHECK_HEADER(pcap.h,HAVE_PCAP_H=true)
+   AC_SUBST(HAVE_PCAP_H)
+fi
+
 AC_SUBST(DATABASE_DIR)
 AC_SUBST(DATABASE_LIB)
 AC_SUBST(DATABASE_LIB_DIR)
@@ -260,7 +272,6 @@
 AC_SUBST(EXTRA_SQLITE3_DEF)
 
 AC_SUBST(DATABASE_DRIVERS)
-AC_SUBST(HAVE_PCAP_H)
 
 AM_CONDITIONAL(HAVE_MYSQL, test x$mysqldir != x)
 AM_CONDITIONAL(HAVE_PGSQL, test x$pgsqldir != x)
